Sources/IndicatorInfo.swift

```diff
@@ -26,21 +26,13 @@ import Foundation
 
 public struct IndicatorInfo {
     
-    public var title: String
+    public var title: String?
     public var image: UIImage?
     public var highlightedImage: UIImage?
     
-    public init(title: String) {
+    public init(title: String? = nil, image: UIImage? = nil, highlightedImage: UIImage? = nil) {
```

Comment:
If we are going to allow having either the title or the image and not both in the `ButtonBarViewCell`, maybe we should have two separated inits: one that recieves a title, and one that recieves the images. This would be in order to stop the user from setting both while using the default `ButtonBarViewCell`. 

If the user wants to use a custom cell provided by him that need other info, then this IndicatorInfo should be subclassed. This changes may require further restructuring, though. What do you think about it?